integration_tests/bindpy_01.py

```diff
@@ -1,11 +1,11 @@
 from lpython import i32, i64, u32, u64, f32, f64, pythoncall
 
 @pythoncall(module = "bindpy_01_module")
-def add_ints(a: i32, b: i64, c: u32, d: u64) -> i64:
+def add_ints(a: u32, b: u32, c: u32, d: u32) -> u64:
     pass
 
 @pythoncall(module = "bindpy_01_module")
-def multiply_ints(a: i32, b: i64, c: u32, d: u64) -> i64:
+def multiply_ints(a: u32, b: u32, c: u32, d: u32) -> u64:
```

Comment:
Are the signed integers tested somewhere? It seems we are now only testing unsigned integers, but we need to test both signed and unsigned.